upup/pkg/fi/cloudup/gcetasks/instancetemplate.go

```diff
@@ -365,10 +373,20 @@ func (e *InstanceTemplate) mapToGCE(project string, region string) (*compute.Ins
 		}
 	}
 
+	var advancedMachineFeatures *compute.AdvancedMachineFeatures
+
+	if e.GCPNestedVirtualization != nil {
+		if advancedMachineFeatures == nil {
+			advancedMachineFeatures = &compute.AdvancedMachineFeatures{}
+		}
+		advancedMachineFeatures.EnableNestedVirtualization = fi.BoolValue(e.GCPNestedVirtualization)
+	}
```

Comment:
Would be nice if we always set this to true or false, to also disable if option is removed from config.
```suggestion
	advancedMachineFeatures &compute.AdvancedMachineFeatures{
		EnableNestedVirtualization: fi.BoolValue(e.GCPNestedVirtualization)
	}
```